Gemfile

```diff
@@ -29,6 +28,7 @@ gem 'jbuilder', '~> 2.7'
 gem 'bootsnap', '>= 1.4.4', require: false
 
 group :development, :test do
+  gem 'sqlite3', '~> 1.4'
```

Comment:
`sqlite3` is put in development because it is not a production-ready database as it does not support multiple users to write the database at the same time

Reply:
So, why not use the database that you will be using in production?

Reply:
From my point of view, the development process might involve changes to the production database that put high risk of losing customers data. The second thing is to protect customer data from developers.

Is there any more reasons?

Reply:
you misunderstand my question.

Production uses postgres.
Why not development use postgres as well?

Reply:
There is no problem in either using `sqlite` in development, `pg` is production or using `pg` for both development and production, it is just whether it is a good practice, whether the hosting platforms like `Heroku` supports the database you chose for production and whether there is a incompatibility issue between development and production (example: if you choose different database for development and production, `sqlite` code works in development and testing, use that code for `pg` for production but does not work and fail to deploy the app).

Reply:
Ok. So, what do you think is the best way to go about this & why?

Reply:
The best way is to stay with only one database (in this case suggest to use `pg` if you want to put it in production) instead of switching between `sqlite` and `pg` for different purposes. This is to ensure consistency of code you use (less mistakes transferring the development code to production code) and reduce the time taken to fix errors if mistakes happened.

Reply:
exactly.
So, pls change this to pg